w3lib/html.py

```diff
@@ -21,6 +21,26 @@
     r'<meta\s[^>]*http-equiv[^>]*refresh[^>]*content\s*=\s*(?P<quote>["\'])(?P<int>(\d*\.)?\d+)\s*;\s*url=\s*(?P<url>.*?)(?P=quote)',
     re.DOTALL | re.IGNORECASE,
 )
+_meta_refresh_re2 = re.compile(
```

Comment:
I like that, but I think we should somehow drop all logic into the same regex pattern